mkdocs.yml

```diff
@@ -2,13 +2,14 @@
 # For details see: https://www.mkdocs.org/user-guide/configuration/
 
 # Project information
-site_name: Frequenz Python SDK
-site_description: Frequenz Python SDK.
-site_author: Frequenz Energy-as-a-Service GmbH
-copyright: Frequenz Energy-as-a-Service GmbH
+site_name: "Frequenz Python SDK"
+site_description: "A development kit to interact with the Frequenz development platform"
+site_author: "Frequenz Energy-as-a-Service GmbH"
+copyright: "Copyright © 2022 Frequenz Energy-as-a-Service GmbH"
```

Comment:
Maybe 2023 as the year it's been just added?

Reply:
No, the file was there before, it wasn't added by this PR.